common-util/src/main/java/org/sunbird/common/models/util/ProjectUtil.java

```diff
@@ -827,6 +848,12 @@ public static String getSMSBody(
       if (StringUtils.isNotBlank(appName)) {
         params.put("appName", appName);
       }
+      params.put(
+          "updatePasswordLink",
```

Comment:
Use constants for updatePasswordLink and verifyEmailLink.

Don't use default values update_password_link and verify_email_link.